package/src/skia/NativeSetup.web.ts

```diff
@@ -0,0 +1 @@
+export const dummy = 0;
```

Comment:
is this needed?

Reply:
makes sense maybe add a comment to it. Or do just keep NativeSetup.ts and skip the process if the Platform.OS === web

Reply:
Done! :) The simplest is often the best!